test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tests Tokenizer in both the quote and no-quote modes.
+ * 
+ * @author (Sarah Abowitz) 
+ * @version (V1.0.0 | 5.20.17)
+ */
+
+package codeu.chat.util;
+
+import static org.junit.Assert.*; 
+
+import java.io.IOException;
+import org.junit.Test; 
+   
+public final class TokenizerTest {
```

Comment:
Nice job adding tests! Consider this to be optional feedback: 

It's usually a good idea to test "edge cases" to make sure your Tokenizer behaves as you'd expect in given interesting inputs. Other things you could test, if you feel so inclined: 
* null input
* an empty string as input
* other types of whitespace characters (such as tabs and newlines) as input.